test/mock.recipe.ts

```diff
@@ -0,0 +1,8 @@
+import { faker } from '@faker-js/faker';
+
+export const recipeData = {
```

Comment:
would be good to use existing type for it - maybe type from Prisma would match? (omitting `id`)

Reply:
Consider creating a factory method that will create a different recipe each time it's called, e.g.:

```typescript
export const createRecipe = (overrides = {}) => ({
  title: overrides.title ?? faker.word.noun(5),
  description: overrides.description ?? faker.lorem.text(),
  ingredients: overrides.ingredients ?? faker.lorem.word(8),
  preparation: overrides.preparation ?? faker.lorem.lines(4),
})
```
or:

```typescript
export const createRecipe = (overrides = {}) => {
  const { title, description, ingredients, preparation } = overrides;

return {
    title: overrides.title ?? faker.word.noun(5),
    description: overrides.description ?? faker.lorem.text(),
    ingredients: overrides.ingredients ?? faker.lorem.word(8),
    preparation: overrides.preparation ?? faker.lorem.lines(4),
  }
}
```

or if you want you can even implement a builder pattern

Reply:
Unfortunately, your code gives me error's `Property <...> does not exist on type {}`
What about this? 
```ts
type createRecipeInput = {
  title?: string;
  description?: string;
  ingredients?: string;
  preparation?: string;
};
export const createRecipe = (overrides: createRecipeInput = {}) => ({
  title: overrides.title ?? faker.word.noun(5),
  description: overrides.description ?? faker.lorem.text(),
  ingredients: overrides.ingredients ?? faker.lorem.word(8),
  preparation: overrides.preparation ?? faker.lorem.lines(4),
});
```

Reply:
I used my own type. Prisma has types such `RecipeUpdateInput` and `RecipeCreateInput`, but nothing in between, which I need to pass it to both `Update` and `Create`. I think it would be weird if I pass request with type `RecipeUpdateInput` to create function and vice versa.

Reply:
I wrote it by hand - I haven't tested it nor included everything there. Yes, interface or type is required, but it should be `PascalCase` not `camelCase`, so `CreateRecipeInput`. Also I don't really like `CreateRecipeInput` name, as it's not input, it's overrides. Maybe `CreateRecipeOverrides`?